core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannelManager.scala

```diff
@@ -301,6 +301,10 @@ class TransactionMarkerChannelManager(config: KafkaConfig,
           // enqueue for retry
           txnLogAppendRetryQueue.add(txnLogAppend)
 
+        case Errors.COORDINATOR_LOAD_IN_PROGRESS =>
```

Comment:
@guozhangwang I think I'm missing something. Isn't it possible that we get the `COORDINATOR_LOAD_IN_PROGRESS` before the CompleteXX is appended to the log in 1)?